winterfell/middlewares/log.js

```diff
@@ -9,7 +9,7 @@ function Log(app) {
   mainLog = bunyan.createLogger({
     name: 'vetafi_' + app.environment,
     streams: [{
-      path: '../logs/app.js', // todo: these logs need to be moved into the same level
+      stream: process.stdout,
```

Comment:
cool we could do it this way. However, when it comes to looking at json objects in a clearer fashion, it's nicer to use the bunyan client. Like so: 
```
tail -F ../logs/app.js | ./node_modules/.bin/bunyan
```

Perhaps we can do the same with pm2?